src/components/Swap/SwapActionButton/AllowanceButton.tsx

```diff
@@ -3,21 +3,17 @@ import { PERMIT2_ADDRESS } from '@uniswap/permit2-sdk'
 import ActionButton from 'components/ActionButton'
 import EtherscanLink from 'components/EtherscanLink'
 import { usePendingApproval } from 'hooks/transactions'
+import useButtonColor from 'hooks/useTokenColorExtraction'
```

Comment:
Run `yarn lint --fix` to get these sorted

Reply:
Be sure to change the imports of `hooks/useTokenColorExtraction` to rename the import to `useTokenExtractionColor`.

Reply:
wow vs code betrayed me with this one